bumps version 1.2.0 (#13)
TotalCross/totalcross-maven-plugin: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.2.0] - 2020-08-05
### Added
- <totalcrossLib> param to configuration. By using this parameter, one can deploy a totalcross application that will take place inside the final jar, i.e., KnowCodeXML.jar has a KnowCodeXMLLib.tcz inside it.

> - capability to load totalcross java libraries that has its tcz files inside it. Using the same example above, when someone adds a library that has its tcz inside it, the plugin is in charge to take this tcz and place inside the final application.

Author: ```suggestion
- Capability of loading TotalCross Java libraries. Using the same example above, when someone adds a library that has its tcz inside it, that is, a TotalCross Java library, now the plugin is charge of copying the library tcz, i.e., `KnowCodeXMLLib.tcz`, and place it inside the final application package.
```

> - parameter <externalResources>. This is a replacement for the file all.pkg. When listing the files inside this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they are accessible inside the final package.

Author: ```suggestion
- parameter <externalResources>: this is a replacement for the file all.pkg. When listing the files inside this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they are accessible inside the final package.
```

    ```xml
    <externalResources>
        <externalResource>my_exposed_file.png</<externalResource>
    </externalResources>    
    ```
### Changed
- Update retrolambda to version 2.5.7. 

### Removed

> 

Author: ```suggestion
```

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.2.0] - 2020-08-05
### Added

> - <totalcrossLib> param to configuration. By using this parameter, one can deploy a totalcross application that will take place inside the final jar, i.e., KnowCodeXML.jar has a KnowCodeXMLLib.tcz inside it.

Author: ```suggestion
- parameter <totalcrossLib>: by setting this parameter to true, one can package a totalcross library that will contain its tcz file inside its final jar, i.e., KnowCodeXML.jar has a KnowCodeXMLLib.tcz inside it.
```

> - capability to load totalcross java librarys that has its tcz files inside it. Using the same example above, when someone adds a library that has its tcz inside it, the plugin is in charge to take this tcz and place inside the final application.

Reviewer: ```suggestion
- capability to load totalcross java libraries that has its tcz files inside it. Using the same example above, when someone adds a library that has its tcz inside it, the plugin is in charge to take this tcz and place inside the final application.
```

> - parameter <externalResources>. This is a replacement for the file all.pkg. When listing the files inside the this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they accessible inside the final package.

Reviewer: ```suggestion
- parameter <externalResources>. This is a replacement for the file all.pkg. When listing the files inside this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they are accessible inside the final package.
```

Reviewer: Didn't understand it well when I read it the first time, I think my suggestion will  make it easier to understand.
